pkgs/by-name/si/signal-desktop-source/webrtc-sources.json

```diff
@@ -0,0 +1,364 @@
+{
+    "src": {
+        "args": {
+            "hash": "sha256-xKv606lqGgC27sf5WVPxkVRI5wixmRxQvOs1+pgORxc=",
+            "owner": "signalapp",
+            "repo": "webrtc",
+            "rev": "6834g"
```

Comment:
Using a short sha here is not suitable for long term and 5 characters can easily cause a (forced) conflict where github will return a 404.